src/IssueUtil.js

```diff
@@ -9,9 +9,11 @@ module.exports = class IssueUtil {
     this.octokit = github.getOctokit(token);
   }
 
-  getIssueBody(id) {
+  getIssueBody(id, repository) {
+    const [owner, repo] = repository.split('/');
```

Comment:
My only issue here is that it is possible for a user to provide an input that is not in this form and we would not detect this of fail nicely with the following call.

I would suggest that we validate the `repository` input early in the execution when we obtain that value from the user inputs and pass a consistent {owner: '', repo: ''} object to this function.